pkg/resources/statefulsets/minio-statefulset.go

```diff
@@ -191,7 +191,7 @@ func minioServerContainer(mi *miniov1.MinIOInstance, serviceName string) corev1.
 
 	if mi.Spec.Zones[0].Servers == 1 {
 		// to run in standalone mode we must pass the path
-		args = append(args, miniov1.MinIOVolumeMountPath)
+		args = append(args, mi.Spec.Mountpath)
```

Comment:
we can perhaps use the helper mi.VolumePath() @dvaldivia

Reply:
agree, that one will actually place the `/export{1...4}` notation, done